src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -138,9 +138,11 @@ def configure(config):
     ]
 
     # don't need an incident context
-    app.command(config.slack_command_list_incidents, middleware=[db_middleware])(
-        handle_list_incidents_command
-    )
+    app.command(
+        config.slack_command_list_incidents,
+        middleware=[db_middleware],
+    )(ack=ack_command, lazy=[handle_list_incidents_command])
```

Comment:
To cut down on duplicate code across the modal `ack` functions, It would be nice if we could inject variables into the `ack=` function (to build the modals).